### PR TITLE
Cleaner Readme.md and small reorg.

### PR DESCRIPTION
**Pull request by etiennesegonzac**
I was able to remove the Makefile for the addon, but we have to make the "b2gdroid" dir a hidden directory otherwise the webide goes bunker.

### CODE REVIEW

**Comment by etiennesegonzac** [comment]
This fixes #13

**Comment by etiennesegonzac** [comment]
r? @wilsonpage 
please nitpick on the Readme :)

File: Readme.md [hunk 1]
```diff
@@ -1,17 +1,44 @@
...
+worked out.
+To guide you, here are some of the design principles used by the team:
+
+* Simplify to the core, focus on the moment.
+* Hierarchy to promote purpose, what's the most important?
+* Add some flare.
+* Big when it's needed.
+
+
+## Memorabilia: The APZ-gambler song
```
Comment (wilsonpage) [inline Readme.md:42]: lol!

File: Readme.md [hunk 2]
```diff
@@ -1,17 +1,44 @@
...
+## Design principles
+We'll have good UX support for this project, but because of the fast
+pace we might have to fill the gap sometimes while final specs are being
+worked out.
+To guide you, here are some of the design principles used by the team:
+
+* Simplify to the core, focus on the moment.
+* Hierarchy to promote purpose, what's the most important?
+* Add some flare.
+* Big when it's needed.
```
Comment (wilsonpage) [inline Readme.md:38]: What about 'fun'? :)

File: manifest.json [hunk 3]
```diff
@@ -36,6 +36,8 @@
     "apps/wired.com/image.png",
     "assets/urlbar_refresh_icon.png",
     "assets/urlbar_reader_mode_icon.png",
+    "assets/Home.png",
+    "assets/Back.png",
```
Comment (wilsonpage) [inline manifest.json:40]: My icon patch removes this dependency. But I can fix that in a rebase :)

File: .b2gdroid/Makefile [hunk 4]
```diff
@@ -4,7 +4,4 @@ clean:
 
 .PHONY: profile
 profile:
-	zip -r -9 profile/webapps/system.gaiamobile.org/application.zip assets index.html lib node_modules js manifest.webapp
-
-addon:
-	zip addon.zip addon.js manifest.json assets index.html lib node_modules
+	zip -r -9 profile/webapps/system.gaiamobile.org/application.zip assets apps index.html lib node_modules js manifest.webapp
```
Comment (wilsonpage) [inline .b2gdroid/Makefile:7]: Is this Makefile script required for the b2gdroid build?

Comment (wilsonpage) [inline .b2gdroid/Makefile:7]: ie. We can't move this script to package.json `"scripts"`?

**Comment by wilsonpage** [comment]
r+

File: .b2gdroid/Makefile (hunk 4 above)
Comment (etiennesegonzac) [inline .b2gdroid/Makefile:7]: yep the b2gdroid build system expects to find this Makefile/task.

Comment (etiennesegonzac) [inline .b2gdroid/Makefile:7]: at least we removed the addon one :)

**Pull request merged**